addon/modifiers/did-update.js

```diff
@@ -58,7 +58,7 @@ import { setModifierManager, capabilities } from '@ember/modifier';
 */
 export default setModifierManager(
   () => ({
-    capabilities: capabilities('3.13', { disableAutoTracking: true }),
```

Comment:
`disableAutoTracking` was intentional (and is what the RFC specifies the behavior of these modifiers should be)